src/services/Email.js

```diff
@@ -0,0 +1,117 @@
+const sendgridClient = require('@sendgrid/mail');
+
+
+const sendCoachEmail = async (userData) => {
+  /* eslint-disable */
+  const {
+    coach_email,
+    coach_name,
+    client_email,
+    client_first_name,
+    client_last_name,
+    client_phone,
+    client_plan_url
+  } = userData;
+
+  const message = {
+    to: coach_email,
+    from: 'support@helloroo.org',
+    subject: 'Complete Workplan',
+    templateId: '21c099ac-1957-4916-9195-83f9bf16dbe9',
+    substitutions: {
+      clientFirstname: client_first_name,
+      clientLastname: client_last_name,
+      clientEmail: client_email,
+      clientPhone: client_phone,
+      clientPlanUrl: client_plan_url,
+      coachFirstName: coach_name
+    },
+  };
+  try {
+    await sendEmail(message);
+  } catch (error) {
+    const message = {
+      to: 'supporst@helloroo.zendesk.com',
+      from: 'no-reply@helloroo.org',
+      subject: `Complete work plan email notification error - ${Date.now()}`,
+      templateId: 'aacf9c28-62be-45ed-a055-feb6aaa95d26',
+      substitutions: {
+        clientFirstname,
+        clientLastname,
+        coach_email,
+        error
+      }
+    }
+    await sendgridClient.send(message);
+  }
+};
+
+const sendHelpPMEmailToCoach = async (userData) => {
+  const {
+    clientId,
+    clientFirstName,
+    clientLastName,
+    coachFirstName,
+    coachEmail,
+    taskTitle,
+    url,
+    steps,
+    helpMessage
+  } = userData;
+
+  const message = {
+    to: coachEmail,
+    from: 'no-reply@helloroo.org',
+    subject: 'Client requests assistance',
+    templateId: 'aacf9c28-62be-45ed-a055-feb6aaa95d26',
+    substitutions: {
+      clientId,
+      clientFirstName,
+      clientLastName,
+      coachFirstName,
+      coachEmail,
+      taskTitle,
+      url,
+      steps,
+      helpMessage
+    },
+  };
```

Comment:
I like what you have done on this task. Please consider adding a blank line after line 78

Reply:
Noted.